Calls ldconfig which is not available and remove -march=native because it's not
portable across platforms.

Upstream: not yet, in touch with the author.
--- a/apps/Makefile	2020-04-17 19:56:50.213695405 +0200
+++ b/apps/Makefile	2020-04-17 19:56:59.230416753 +0200
@@ -55,8 +55,8 @@
 
 
 install:	all
-	install -d $(BINDIR)
-	install -d $(MANDIR)
+	install -d $(DESTDIR)$(BINDIR)
+	install -d $(DESTDIR)$(MANDIR)
 	install -m 755 zresample $(DESTDIR)$(BINDIR)
 	install -m 755 zretune $(DESTDIR)$(BINDIR)
 	install -m 644 zresample.1.gz $(DESTDIR)$(MANDIR)
--- a/source/Makefile	2020-04-19 10:26:26.758860827 +0200
+++ b/source/Makefile	2020-04-19 10:26:38.365583098 +0200
@@ -33,7 +33,6 @@
 
 CPPFLAGS += -I. -D_REENTRANT -D_POSIX_PTHREAD_SEMANTICS
 CXXFLAGS += -Wall -fPIC -O2 -ffast-math
-CXXFLAGS += -march=native
 LDFLAGS += 
 LDLIBS +=
 
@@ -58,7 +57,6 @@
 	install -d $(DESTDIR)$(LIBDIR)
 	install -m 644 $(ZITA-RESAMPLER_H)   $(DESTDIR)$(INCDIR)/zita-resampler
 	install -m 755 $(ZITA-RESAMPLER_MIN) $(DESTDIR)$(LIBDIR)
-	ldconfig
 	ln -sf $(ZITA-RESAMPLER_MIN) $(DESTDIR)$(LIBDIR)/$(ZITA-RESAMPLER_SO)
 
 uninstall:
